com.liferay.blade.cli/src/com/liferay/blade/cli/CreateCommand.java

```diff
@@ -244,7 +244,7 @@ else if ("activator".equals(template)) {
 
 		if (isWorkspace) {
 			final Pattern buildGlob = Pattern.compile(
-				"^workspace/" + template + "/.*|\\...+/build.gradle");
+				"^workspace/" + template + "/.*|\\...+/.*");
```

Comment:
I see, this just adds everything under workspace template assuming that it may have a "_service_" there now.